brother_ql/cli.py

```diff
@@ -49,16 +49,15 @@ def discover(ctx):
     if backend is None:
         logger.info("Defaulting to pyusb as backend for discovery.")
         backend = "pyusb"
-    from brother_ql.backends.helpers import discover, status
+    from brother_ql.backends.helpers import discover, get_printer, get_status
 
     available_devices = discover(backend_identifier=backend)
     for device in available_devices:
-        device_status = None
-        result = {"model": "unknown"}
+        status = None
 
         # skip network discovery since it's not supported
         if backend == "pyusb" or backend == "linux_kernel":
-            logger.info(f"Probing device at {device['identifier']}")
+            print(f"Probing device at {device['identifier']}")
```

Comment:
Q (blocking): Is there a reason why this was replaced? In my experience `print` is pretty hard to handle and test reliably compared with `logger.{level}`

Reply:
Same as above.

Reply:
Would you be open to rethink this? `print` is very hard to get rid off while log levels can easily be filtered by setting approriate enviroment variables.
My thinking here is to not make the life of ppl automating stuff via the cli harder than it has to be.